packages/next/src/build/webpack/plugins/flight-manifest-plugin.ts

```diff
@@ -296,9 +312,24 @@ export class ClientReferenceManifestPlugin {
         /[\\/]/g,
         path.sep
       )
+
       manifest.entryCSSFiles[chunkEntryName] = entrypoint
         .getFiles()
         .filter((f) => !f.startsWith('static/css/pages/') && f.endsWith('.css'))
+        .map((file) => {
+          const source = compilation.assets[file].source()
+          if (this.experimentalInlineCss) {
```

Comment:
I think this is were the dev/build condition (using `this.dev`) should be added (and the equivalent in Turbopack), not it `packages/next/src/server/app-render/render-css-resource.tsx`.